R/reexports.R

```diff
@@ -0,0 +1,7 @@
+#' @export
+#' @importFrom effectsize standardize
+effectsize::standardize
```

Comment:
can't we import them directly from where they are?

Reply:
Oh you mean this is so that users can use standardize() directly without library(effectsize) is it?

Reply:
yes, it's a re-export

Reply:
Huh, this was the solution but somehow the PR builds still failed. Anyway, doesn’t matter. Seems to be fixed in the master by now.